README.md

```diff
@@ -6,12 +6,14 @@ This is the unofficial scss version of Bulma.  For all questions or concerns abo
 
 Bulma is a **modern CSS framework** based on [Flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Using_CSS_flexible_boxes).
 
-[![npm](https://img.shields.io/npm/v/bulma_scss.svg)](https://www.npmjs.com/package/bulma_scss)
-[![npm](https://img.shields.io/npm/dm/bulma_scss.svg)](https://www.npmjs.com/package/bulma_scss)
+[![npm](https://img.shields.io/npm/v/bulma.svg)][npm-link]
```

Comment:
@illogic-al Can you please update this to this project's badges?

Reply:
How would I do that? I don't know what those are?

Reply:
Well, I'm not sure if you are familiar with pull requests, however all diffs can be seen here.  This is what I am reviewing.  So you can see the lines above that you replaced point to bulma_scss badges, and you replaced them with bulma badges.  Please undo this change

Reply:
Ah, I see. Replaced your project's images (badges) with the bulimia project's. Yup, I can fix that.

Reply:
OK. Changes have been pushed.